chef/automate
# document external facing reporting stats proto

### :nut_and_bolt: Description: What code changed, and why?

Documenting the external facing stats proto. 

### :chains: Related Resources
fixes https://github.com/chef/automate/issues/2018

### :+1: Definition of Done
stats proto is documented

### :athletic_shoe: How to Build and Test the Change
- check out this branch
- run make serve from the automate-chef-io directory
- visit http://localhost:1313/docs/api
